changelogs/unreleased/gh-10347-proper-instance-duplicates-messages.md

```diff
@@ -0,0 +1,5 @@
+## bugfix/config
+
+* Now Tarantool writes a detailed error message if it finds
+  groups/replicasets/instances with the same name in the provided
+  configuration (gh-10347).
```

Comment:
```suggestion
  groups, replica sets, or instances with the same name in the provided
  configuration (gh-10347).
```

Reply:
Thank you.  Though I had to change the message a bit. Please, check it out:
```
## bugfix/config

* Now Tarantool writes a detailed error message if it finds
  replica sets with the same names in different groups or instances
  with the same names in different replica sets in the provided
  configuration (gh-10347).
```